contracts/sophia.md

```diff
@@ -546,6 +546,15 @@ The shortest unique suffix could also be used, and the hash could be ANDed with
 Then each exported function starts with an exported entry point where it executes
 code to fetch the arguments to memory.
 
+### Contract state
+
+Before a contract call is executed the current state of the contract is loaded
+into memory encoded as described above. A pointer to the encoded state is stored at
+address 0. The Sophia compiler generates code to unpack the encoded state and
```

Comment:
Could you please confirm address 0 in other places - I found:
https://github.com/aeternity/protocol/pull/67/commits/84d3b0931235e394715ce762cb2fce68f2c33835#diff-9ddeae469112ac9dd44768c45511c6e9R527
https://github.com/aeternity/protocol/pull/67/commits/84d3b0931235e394715ce762cb2fce68f2c33835#diff-9ddeae469112ac9dd44768c45511c6e9R528

Reply:
Ah, well spotted. Those should be addr 32.